src/api/schemas/user-schema.ts

```diff
@@ -141,4 +170,15 @@ const submitQuiz = object({
 });
 
 export { firebase_id, firstName, lastName, email, phone, countryCode };
-export { userSchema, createUserSchema, getAllUsers, getUserByEmailSchema, getUserById, getUserActivityList, updateUserSchema, addNewUserActivity, submitQuiz };
+export {
+    userSchema,
+    createUserSchema,
+    getAllUsers,
+    getUserByEmailSchema,
+    getUserById,
+    getUserActivityList,
+    updateUserSchema,
+    userAddFavoriteActivitySchema as userFavoriteActivityListSchema,
```

Comment:
not... really necessary heh. just keep the old name and use it in `validate(...)`

Reply:
this was left here by mistake. Fixed!